documentation/analytics/recipes/player-geo-cluster-analysis.md

```diff
@@ -162,14 +162,14 @@ Task.Run(async () =>
 
 | Query that should be run           | By what service                           |
 |------------------------------------|-------------------------------------------|
```

Comment:
Thanks for cleaning up this file. Looks good!

I'd approve these changes right away, but since this file is in the same PR as the below retention file, I can't do it quite yet.